rasa/core/policies/ted_policy.py

```diff
@@ -604,7 +656,11 @@ def persist(self, path: Text) -> None:
             model_path / f"{SAVE_MODEL_FILE_NAME}.data_example.pkl", self.data_example
         )
         io_utils.pickle_dump(
-            model_path / f"{SAVE_MODEL_FILE_NAME}.label_data.pkl", self._label_data
+            model_path / f"{SAVE_MODEL_FILE_NAME}.zero_features.pkl", self.zero_features
```

Comment:
is `self.zero_features` is basically equivalent to `self.data_example` but with zeros?

Reply:
No. It might be that the first `data_example` does not have all attributes present. That would result in issue. I think it is best to store the `zero_features` and to load them again.

Reply:
ok, but let's update the name